Libraries/Animated/src/AnimatedImplementation.js

```diff
@@ -2187,7 +2187,13 @@ function attachNativeEvent(viewRef: any, eventName: string, argMapping: Array<?M
 
   return {
     detach() {
-      NativeAnimatedAPI.removeAnimatedEventFromView(viewTag, eventName);
+      eventMappings.forEach((mapping) => {
+        NativeAnimatedAPI.removeAnimatedEventFromView(
+          viewTag,
+          eventName,
+          mapping.animatedValueTag,
```

Comment:
This extra arg will just get ignored if the binary has the old API right? I'd like this keep working if people update the JS ahead of the native code if possible.